packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -86,6 +87,7 @@ export const styles = theme => ({
 const ListItem = React.forwardRef(function ListItem(props, ref) {
   const {
     alignItems,
+    autoFocus,
```

Comment:
I believe we have discussed the introducing of this property to the MenuItem component. What has changed? @mbrookes Has initiated a related effort in #15334.

Reply:
If you're asking what is the motivation for putting it in `ListItem` instead of `MenuItem`, I just figured it made sense to put the functionality closer to the rendering of the DOM node since `MenuItem` leverages `ListItem` unconditionally. Currently `MenuItem` has almost zero functionality of its own and mainly just adds some styling changes to `ListItem`, so this continues that pattern of letting `ListItem` do the bulk of the work. I don't have strong feelings about it, but I wasn't seeing any downside to putting it in `ListItem`.

Reply:
I'm wondering if we shouldn't move the focus logic one more layer down to the ButtonBase component. It's fine here for now :).